src/020._valid_parentheses.md

```diff
@@ -67,4 +69,40 @@ class Solution(object):
                 if stack == [] or stack.pop() != dic[char]:
                     return False
         return stack == []
-```
\ No newline at end of file
+```
+
+### C++ Solutions
+
+* Time complexity : O(n)
+* Space complexity : O(n)
+
+```C++
+class Solution {
+public:
+    bool isValid(string s) {
```

Comment:
const string &s

Reply:
I understand your meaning,but this is what the problem shows. I just copy the function name and write the function body.

Reply:
OK, I see.